lighthouse-core/config/default.js

```diff
@@ -199,6 +209,10 @@ module.exports = {
       title: 'Meta Tags Used Properly',
       description: 'These are opportunities to improve the user experience of your site.',
     },
+    'manual-a11y-checks': {
+      title: 'Manual checks to verify',
```

Comment:
i know you were just copying what we already had but....  wdyt about a diff title for these?

like

> Additional checks to manually verify

> Additional items to manually check

Reply:
sgtm